src/reasonJs.re

```diff
@@ -2,11 +2,13 @@ external requestAnimationFrame : (unit => unit) => unit = "" [@@bs.val];
 
 module Base64 = Base64Re;
 module Gl = GlRe;
+module Canvas2d = Canvas2d;
 
 module Storage = StorageRe;
 
 module CanvasElement = {
-  external getContext : Dom.element => string => Gl.glT = "" [@@bs.send];
+  external getWebglContext : Dom.element => (_ [@bs.as "webgl"]) => Gl.glT = "getContext" [@@bs.send];
+  external get2dContext : Dom.element => (_ [@bs.as "2d"]) => Canvas2d.ctx = "getContext" [@@bs.send];
```

Comment:
I'm wondering if these shouldn't be `getContext2d` and `getContextWebGl` to aid in discoverability.